dev-utils/_windows/breakpad-tools/breakpad-tools.py

```diff
@@ -12,14 +12,15 @@ def registerOptions(self):
     """We use Jon Turney's fork which adds support for MinGW and a python script to fetch deps"""
     def setTargets(self):
         for ver in ["pecoff-dwarf-on-git-20171117-fetch-externals-on-win"]:
-            self.svnTargets[ver] = f"[git]https://github.com/dschmidt/google-breakpad|{ver}|"
+            self.svnTargets[ver] = f"[git]https://github.com/dschmidt/google-breakpad|patch-1|"
```

Comment:
nitpicking but why don't you add the patch to the list and change the default target ^^

Reply:
So copy-pasting is easier when I revert ;D

Reply:
ah, I see .. wait